ketchsdk/src/main/java/com/ketch/android/Ketch.kt

```diff
@@ -186,7 +186,10 @@ class Ketch private constructor(
                 }
 
                 override fun onConfigUpdated(config: KetchConfig?) {
+                    // Set internal config field
                     this.config = config
+                    // Call config update listener
+                    this@Ketch.listener?.onConfigUpdated(config)
```

Comment:
typo in the `@`?

Reply:
@prachikhandekar nope, that's just _kotlin_ 🥂